src/tests/ftest/io/mdtest_small.yaml

```diff
@@ -0,0 +1,57 @@
+hosts:
+    test_servers:
+        - boro-A
+        - boro-B
+        - boro-C
+        - boro-D
+    test_clients:
+        - boro-E
+        - boro-F
+timeout: 600
+server_config:
+    name: daos_server
+pool:
+    createmode:
+        mode_RW:
+             mode: 146
+    createset:
+        setname: daos_server
+    createsize:
+        scm_size: 14000000000
+    createsvc:
+        svcn: 1
+mdtest:
+    client_processes:
+        np_8:
+            np: 8
+    api: 'DFS'
+    test_dir: "/"
+    iteration: 1
+    data: !mux
+        zero:
+            write_bytes: 0
+            read_bytes: 0
+        4K:
+            write_bytes: 4096
+            read_bytes: 4096
+    branch: !mux
+        zero:
+            branching_factor: 1
+            num_of_files_dirs: 1000
+            diff_flags: !mux
+                flags_ssf:
+                    flags: "--dfs.destroy"
+                flags_fpp:
+                    flags: "-u --dfs.destroy"
+            directory_depth: !mux
+                depth_0:
+                    depth: 0
+                depth_20:
+                    depth: 20
+        two:
+            branching_factor: 2
+            items: 10
+            flags: "--dfs.destroy"
+            depth: 5
+    dfs_cont: "`uuidgen`"
```

Comment:
I believe you don't need this as the `set_daos_params` method will set a unique uuid if the value is not assigned.